apps/snippets/models.py

```diff
@@ -96,10 +96,12 @@ class NavigationMenu(ClusterableModel):
         max_length=255,
         null=False,
         blank=False,
-        help_text=_('Call the menu "header" to use as upper menu. '
-                    'Call the menu "footer" to use as'
-                    'the upper light grey footer. The lower '
-                    'footer is loaded externally.')
+        help_text=_('Call the menu "header" for use on de and en pages '
+                   'and header_ls for ls pages as the upper menu. '
```

Comment:
really nit-picking, but could we also add quotation marks here (also for footer) to be consistent?